### ISSUE TITLE
[update] Uncaught WebDriverError: Session ID is null. Using WebDriver after calling quit()?- issue 

### ISSUE THREAD

**Issue by lisettea** [Codeception/CodeceptJS#602]
So I've been running into this issue for a while and it seems related to session buildup and teardown? I was hoping someone could point me in the direction of a potential fix - or who could possibly help me get to the root of the problem/create a fix. The issue stands as follows:

#### What are you trying to achieve?

To run tests in parallel or sequentially.  

#### What do you get instead?

For parallel testing: 
 - When one test fails (Chrome) and its session is closed, other sessions which are running at the same time (IE) are closed too. 

For sequential testing:
When one feature file has succesfully (or indeed unsuccesfully) completed their run and tears down it's session, the second feature file's session is built during the teardown of the first session. Somehowt this causes the second session to crash as well - and there seems to be no sessionId generated for it in the first place. 

It's my belief it might something to do with the session buildup/teardown? The selenium server output (from below) shows the second session being built while the first session teardown is still executing. 

### codeceptjs output
> codeceptjs run-multiple --all --reporter mochawesome-screenshots 

[mochawesome] Generating report files...

  WOON
    1) zo snel mogelijk door de gehele funnel, MAXIMAAL alles vullen

  WOON
    2) zo snel mogelijk door de gehele funnel, MINIMAAL alles vullen

  0 passing (18s)
  2 failing

  1) WOON: zo snel mogelijk door de gehele funnel, MAXIMAAL alles vullen:

      expected element //div[@class= 'card__content'] to include "28,33"
      + expected - actual

  Scenario Steps:

  - I.click("#forwardNav") at Test.Scenario (tests\Uitvoeren\WoonfunnelKlikMax.j                                                                                 s:109:7)
  - I.saveScreenshot("WoonAliveMAX_Stap5naam.jpg") at Test.Scenario (tests\Uitvo                                                                                 eren\WoonfunnelKlikMax.js:107:7)
 [...]

  2) WOON: zo snel mogelijk door de gehele funnel, MINIMAAL alles vullen:
     Uncaught WebDriverError: Session ID is null. Using WebDriver after calling                                                                                  quit()?
Build info: version: '3.4.0', revision: 'unknown', time: 'unknown'
System info: host: 'MS600397-SP1004', ip: '145.219.224.91', os.name: 'Windows Se                                                                                 rver 2008 R2', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_131'
Driver info: driver.version: RemoteWebDriver
      at WebDriverError (node_modules\selenium-webdriver\lib\error.js:27:5)
      at Object.checkLegacyResponse (node_modules\selenium-webdriver\lib\error.j                                                                                 s:505:15)
      at parseHttpResponse (node_modules\selenium-webdriver\lib\http.js:509:13)
      at doSend.then.response (node_modules\selenium-webdriver\lib\http.js:440:1                                                                                 3)
      at process._tickCallback (internal/process/next_tick.js:109:7)
  From: Task: WebDriver.manage().deleteAllCookies()
      at thenableWebDriverProxy.schedule (node_modules\selenium-webdriver\lib\we                                                                                 bdriver.js:816:17)
      at Options.deleteAllCookies (node_modules\selenium-webdriver\lib\webdriver                                                                                 .js:1276:25)
      at Protractor._after (node_modules\codeceptjs\lib\helper\SeleniumWebdriver                                                                                 .js:151:27)
      at recorder.add (node_modules\codeceptjs\lib\listener\helpers.js:23:69)

```
### Selenium server code

```Selenium Output
10:52:49.586 INFO - Done: [take screenshot]
10:52:49.630 INFO - Executing: [take screenshot])
10:52:49.637 INFO - Executing: [delete session: c12b5113-2ee2-482f-b574-b5353a66
8670])
10:52:49.638 INFO - Executing: [delete all cookies])
10:52:49.641 INFO - Executing: [new session: Capabilities [{browserName=chrome}]
])
10:52:49.643 INFO - Creating a new session for Capabilities [{browserName=chrome
}]
Starting ChromeDriver 2.29.461591 (62ebf098771772160f391d75e589dc567915b233) on
port 6867
Only local connections are allowed.
10:52:49.969 INFO - Done: [take screenshot]
10:52:50.969 INFO - Done: [delete session: c12b5113-2ee2-482f-b574-b5353a668670]

10:52:50.985 WARN - Exception thrown
org.openqa.selenium.NoSuchSessionException: Session ID is null. Using WebDriver
after calling quit()?
```
### Details

* CodeceptJS version: 0.6.3
* NodeJS Version: 3.10.10
* Operating System: Windows
* Protractor
* Configuration file:

```js
exports.config = {

   tests: 'tests/*.js',
    timeout: 10000,
    output: './output',
    multiple: {
        'basic': {
            'browsers': ['chrome', 'internet explorer']
        }
    },

    helpers: {
    Protractor: {
        'seleniumAddress': 'http://localhost:4446/wd/hub',
        'url': process.env.URL',
        'browser': process.env.BROWSER                  
    },

    Iframe: {
        require: './helpers/iframe_helper.js'
    }
},

  include: createIncludeObject(),

        
       mocha: {
        reporter: 'mochawesome',
        reporterOptions: {
        reportDir: 'report',
        reportTitle: 'chromereport',
        enableCode: true        
    }
},
    name: 'Centraal Beheer'
}
```

**Comment by lisettea** [comment]
Replying to this: Downgraded to codecept 0.6.2 and this seems to fix the issue. My conclusion is therefore that something was implemented in 0.6.3 which causes this.

**Comment by lisettea** [comment]
Fixed in release 1.0.0! Thanks :dancing_women:

**Comment by vidyashreenagashetty** [comment]
I am facing same issue how to resolve this ... why codeconcept coming in webdriver I didn't understand can some body suggest
